src/js/components/app.coffee

```diff
@@ -95,6 +96,12 @@ App = React.createClass
       if data.activity_url.match(SEQUENCE_ID_REGEXP)
         sequenceId= data.activity_url.match(SEQUENCE_ID_REGEXP)[1]
 
+      @logManager.log
+        event: "setOffering"
```

Comment:
Not a big deal, but I guess 'setOffering' means that we just updated report, right? Maybe we could use more meaningful name for report users. Also, it happens automatically every X seconds, so I'd wonder if we really want to log it.